features/admin.actions.v1/pages/actions.tsx

```diff
@@ -82,12 +88,106 @@ export const ActionTypesListingPage: FunctionComponent<ActionTypesListingPageInt
     const { getLink } = useDocumentation();
     const dispatch: Dispatch = useDispatch();
 
+    const { isSubOrganization } = useGetCurrentOrganizationType();
     const {
         data: actionTypesConfigs,
         isLoading: isActionTypesConfigsLoading,
         error: actionTypesConfigsRetrievalError
     } = useGetActionTypes();
 
+    const isActionTypeDisabled = (actionType: string): boolean =>
+        actionsFeatureConfig["featureFlags"]?.some(
+            (featureFlag: FeatureFlagsInterface) => featureFlag.feature === actionType
+            && featureFlag.flag === ActionsConstants.ACTION_COMING_SOON_LABEL
+        );
```

Comment:
To check disabled features, we need to check the `disabledFeatures` array, not `featureFlags`